Android/Kotlin/samples/KotlinSampleLibrary/KotlinSampleLibrary.csproj

```diff
@@ -42,7 +42,7 @@
     <Reference Include="Mono.Android" />
   </ItemGroup>
   <ItemGroup>
-    <PackageReference Include="Xamarin.Kotlin.StdLib" Version="1.4.20" />
+    <PackageReference Include="Xamarin.Kotlin.StdLib.Jdk7" Version="1.4.21" />
```

Comment:
Why was this change needed? Do we no longer need the stdlib?

Reply:
`Xamarin.Kotlin.StdLib.Jdk7` has dependency to `Xamarin.Kotlin.StdLib`